### PR TITLE
Sign back in to last team on relaunch

### PR DESCRIPTION
**Pull request by canuc**
fix for issue https://github.com/wlaurance/slack-for-linux/issues/4. 

Whenever we get a 'document-start' (https://github.com/rogerwang/node-webkit/wiki/Window#document-start) event we cache the sub-domain in localStorage. If the domain is a plain slack domain we assume a logout. Upon start-up we append the iframe with the cached sub-domain as the source url.

### CODE REVIEW

**Comment by canuc** [comment]
@wlaurance @twolfson Hey, fixed the jscs issues should be good to merge!

File: app/js/index.js [hunk 1]
```diff
@@ -13,6 +23,55 @@
 		}
 	});
 
+	function newLocationToProcess(locationToProcess) {
+		var locationHostname = urllib.parse(locationToProcess).hostname;
+
+		if (validSlackSubdomain.test(locationHostname)) {
+			var subdomain = validSlackSubdomain.exec(locationHostname);
+			if (subdomain[1] && subdomain[1].length > 1) {
```
Comment (twolfson) [inline app/js/index.js]: The `test` is redundant to the `exec`. We can remove the first `if` check

File: app/js/index.js [hunk 2]
```diff
@@ -13,6 +23,55 @@
...
+		}
+	}
+
+	function handleLoadIframe(url) {
+		var bodyElement = document.body;
+
+		// Remove all the body's children nodes
+		while (bodyElement.firstChild) {
+			bodyElement.removeChild(bodyElement.firstChild);
+		}
```
Comment (twolfson) [inline app/js/index.js]: The comment here explains what is happening but still gives no insight into *why* we are doing it.

**Comment by twolfson** [comment]
Seems fine. My issues are mostly pedantic. I will leave it up to @wlaurance.

**Pull request merged**